src/nix-store/nix-store.cc

```diff
@@ -808,14 +808,13 @@ static void opServe(Strings opFlags, Strings opArgs)
         if (GET_PROTOCOL_MINOR(clientVersion) >= 2)
             settings.maxLogSize = readNum<unsigned long>(in);
         if (GET_PROTOCOL_MINOR(clientVersion) >= 3) {
-            settings.buildRepeat = readInt(in);
+            readInt(in); // this used to be build-repeat, but it never worked anyway
```

Comment:
I think no longer supported `buildRepeat` values should be an error, just as `--repeat` on the CLI will become an error.